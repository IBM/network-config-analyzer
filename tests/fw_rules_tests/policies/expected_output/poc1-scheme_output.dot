// The Connectivity Graph of poc1
digraph poc1 {
	HEADER [shape="box" label=< <B>connectivity_map_dot/poc1</B> > fontsize=30 color=webmaroon fontcolor=webmaroon];
	"0.0.0.0/0" [label="0.0.0.0/0" color="red2" fontcolor="red2"]
	"::/0" [label="::/0" color="red2" fontcolor="red2"]
	"default/adservice-1" [label="default/adservice-1" color="blue" fontcolor="blue"]
	"default/cartservice-1" [label="default/cartservice-1" color="blue" fontcolor="blue"]
	"default/checkoutservice-1" [label="default/checkoutservice-1" color="blue" fontcolor="blue"]
	"default/currencyservice-1" [label="default/currencyservice-1" color="blue" fontcolor="blue"]
	"default/emailservice-1" [label="default/emailservice-1" color="blue" fontcolor="blue"]
	"default/frontend-1" [label="default/frontend-1" color="blue" fontcolor="blue"]
	"default/loadgenerator-1" [label="default/loadgenerator-1" color="blue" fontcolor="blue"]
	"default/paymentservice-1" [label="default/paymentservice-1" color="blue" fontcolor="blue"]
	"default/productcatalogservice-1" [label="default/productcatalogservice-1" color="blue" fontcolor="blue"]
	"default/recommendationservice-1" [label="default/recommendationservice-1" color="blue" fontcolor="blue"]
	"default/redis-cart-1" [label="default/redis-cart-1" color="blue" fontcolor="blue"]
	"default/shippingservice-1" [label="default/shippingservice-1" color="blue" fontcolor="blue"]
	"0.0.0.0/0" -> "default/frontend-1" [label=" TCP, 8080" color="gold2" fontcolor="darkgreen"]
	"::/0" -> "default/frontend-1" [label=" TCP, 8080" color="gold2" fontcolor="darkgreen"]
	"default/cartservice-1" -> "default/redis-cart-1" [label=" TCP, 6379" color="gold2" fontcolor="darkgreen"]
	"default/checkoutservice-1" -> "default/cartservice-1" [label=" TCP, 7070" color="gold2" fontcolor="darkgreen"]
	"default/checkoutservice-1" -> "default/currencyservice-1" [label=" TCP, 7000" color="gold2" fontcolor="darkgreen"]
	"default/checkoutservice-1" -> "default/emailservice-1" [label=" TCP, 8080" color="gold2" fontcolor="darkgreen"]
	"default/checkoutservice-1" -> "default/paymentservice-1" [label=" TCP, 50051" color="gold2" fontcolor="darkgreen"]
	"default/checkoutservice-1" -> "default/productcatalogservice-1" [label=" TCP, 3550" color="gold2" fontcolor="darkgreen"]
	"default/checkoutservice-1" -> "default/shippingservice-1" [label=" TCP, 50051" color="gold2" fontcolor="darkgreen"]
	"default/frontend-1" -> "default/adservice-1" [label=" TCP, 9555" color="gold2" fontcolor="darkgreen"]
	"default/frontend-1" -> "default/cartservice-1" [label=" TCP, 7070" color="gold2" fontcolor="darkgreen"]
	"default/frontend-1" -> "default/checkoutservice-1" [label=" TCP, 5050" color="gold2" fontcolor="darkgreen"]
	"default/frontend-1" -> "default/currencyservice-1" [label=" TCP, 7000" color="gold2" fontcolor="darkgreen"]
	"default/frontend-1" -> "default/productcatalogservice-1" [label=" TCP, 3550" color="gold2" fontcolor="darkgreen"]
	"default/frontend-1" -> "default/recommendationservice-1" [label=" TCP, 8080" color="gold2" fontcolor="darkgreen"]
	"default/frontend-1" -> "default/shippingservice-1" [label=" TCP, 50051" color="gold2" fontcolor="darkgreen"]
	"default/loadgenerator-1" -> "default/frontend-1" [label=" TCP, 8080" color="gold2" fontcolor="darkgreen"]
	"default/recommendationservice-1" -> "default/productcatalogservice-1" [label=" TCP, 3550" color="gold2" fontcolor="darkgreen"]
}


