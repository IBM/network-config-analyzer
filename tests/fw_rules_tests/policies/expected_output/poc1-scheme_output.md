|query|src_ns|src_pods|dst_ns|dst_pods|connection|
|---|---|---|---|---|---|
|connectivity_map, config: poc1||||||
||[default]|[app in (checkoutservice,frontend,recommendationservice)]|[default]|[productcatalogservice]|TCP 3550,|
||[default]|[app in (checkoutservice,frontend)]|[default]|[shippingservice]|TCP 50051,|
||[default]|[checkoutservice]|[default]|[paymentservice]|TCP 50051,|
||[default]|[frontend]|[default]|[checkoutservice]|TCP 5050,|
||[default]|[cartservice]|[default]|[redis-cart]|TCP 6379,|
||[default]|[app in (checkoutservice,frontend)]|[default]|[currencyservice]|TCP 7000,|
||[default]|[app in (checkoutservice,frontend)]|[default]|[cartservice]|TCP 7070,|
|||ip block: 0.0.0.0/0|[default]|[frontend]|TCP 8080,|
||[default]|[checkoutservice]|[default]|[emailservice]|TCP 8080,|
||[default]|[frontend]|[default]|[recommendationservice]|TCP 8080,|
||[default]|[loadgenerator]|[default]|[frontend]|TCP 8080,|
||[default]|[frontend]|[default]|[adservice]|TCP 9555,|


