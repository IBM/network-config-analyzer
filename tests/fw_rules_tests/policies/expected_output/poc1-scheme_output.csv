"query","src_ns","src_pods","dst_ns","dst_pods","connection",
"connectivity_map_csv, config: poc1","","","","","",
"","[default]","[app in (checkoutservice,frontend,recommendationservice)]","[default]","[productcatalogservice]","{protocols:TCP,dst_ports:3550}",
"","[default]","[checkoutservice]","[default]","[app in (paymentservice,shippingservice)]","{protocols:TCP,dst_ports:50051}",
"","[default]","[frontend]","[default]","[shippingservice]","{protocols:TCP,dst_ports:50051}",
"","[default]","[frontend]","[default]","[checkoutservice]","{protocols:TCP,dst_ports:5050}",
"","[default]","[cartservice]","[default]","[redis-cart]","{protocols:TCP,dst_ports:6379}",
"","[default]","[app in (checkoutservice,frontend)]","[default]","[currencyservice]","{protocols:TCP,dst_ports:7000}",
"","[default]","[app in (checkoutservice,frontend)]","[default]","[cartservice]","{protocols:TCP,dst_ports:7070}",
"","","0.0.0.0/0","[default]","[frontend]","{protocols:TCP,dst_ports:8080}",
"","[default]","[checkoutservice]","[default]","[emailservice]","{protocols:TCP,dst_ports:8080}",
"","[default]","[frontend]","[default]","[recommendationservice]","{protocols:TCP,dst_ports:8080}",
"","[default]","[loadgenerator]","[default]","[frontend]","{protocols:TCP,dst_ports:8080}",
"","[default]","[frontend]","[default]","[adservice]","{protocols:TCP,dst_ports:9555}",
